app/main/posts/views/mode-context.html

```diff
@@ -17,5 +17,5 @@ <h1 class="mode-context-title"><a href="/" ng-bind="nav.site.name"></a></h1>
         <!-- <saved-search-create></saved-search-create> -->
         <mode-context-form-filter></mode-context-form-filter>
     </div>
-    <my-component foo=3 baz=2></my-component>
+    <my-component store={store} foo=3 baz=2></my-component>
```

Comment:
I missed this - but we shouldn't need to pass in store here. It might break things